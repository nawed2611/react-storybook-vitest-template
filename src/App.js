import { Button } from './stories/Button'
import { Header } from './stories/Header'

function App() {
  return (
    <div className="App">
      <Header
        onCreateAccount={() => { }}
        onLogin={function noRefCheck() { }}
        onLogout={function noRefCheck() { }}
        user={{
          name: 'Nawed Ali'
        }}
      />
      <a href="/about">About</a>
      <Button
        label="Submit"
        onClick={() => { }}
        primary
      />
    </div>
  );
}

export default App;
