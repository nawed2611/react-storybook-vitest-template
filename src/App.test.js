import { render, screen } from '@testing-library/react';
import App from './App';

test('renders button with name', () => {
  render(<App />);
  const linkElement = screen.getByText(/Submit/i);
  expect(linkElement).toBeInTheDocument();
});
