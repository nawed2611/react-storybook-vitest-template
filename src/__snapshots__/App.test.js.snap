// Vitest Snapshot v1

exports[`renders correctly 1`] = `undefined`;
